.github/workflows/tests.yml

```diff
@@ -0,0 +1,34 @@
+name: ✔💢 Python tests and SonarQube
+
+on:
+  push:
+    branches:
+      - current
+      - next
+  pull_request:
+    types: [opened, synchronize, reopened]
```

Comment:
Isnt this just the default? Added for clearity?

Reply:
Yes, so that I also learn me some defaults 😆